evalml/automl/automl_search.py

```diff
@@ -258,7 +253,7 @@ def __init__(self,
         self.allowed_pipelines = allowed_pipelines
         self.allowed_model_families = allowed_model_families
         self._automl_algorithm = None
-        self._start = None
+        self._start = 0.0
```

Comment:
Its weird but if this is initialized as `None`, it'd cause `add_to_rankings` to error out during logging if the search hasn't started yet.